autocleaner.py

```diff
@@ -91,20 +91,71 @@ def load_yaml(yml_path):
         sys.exit(1)
 
 
-def delete_old_job_dirs(max_days):
+def get_old_job_dirs(max_days):
     """
-    Delete all PRCI job directories older then max_days if "--jobs_dir_exp"
+    Get all PRCI job directories older than max_days if "--jobs_dir_exp"
     cmdline argument defined
     """
-    dirs_deleted = []
+    old_job_dirs = []
     max_days = datetime.timedelta(max_days)
     for folder in os.scandir(JOBS_DIR):
         if folder.is_dir():
             mtime = datetime.datetime.fromtimestamp(
                 os.path.getmtime(folder.path))
             if mtime - datetime.datetime.now() < max_days:
-                shutil.rmtree(folder.path)
-                dirs_deleted.append(folder.path)
+                old_job_dirs.append(folder.path)
+    return old_job_dirs
+
+
+def prune_exports_file(prune_dirs):
+    """
+    Vagrant entries in /etc/exports are not removed on vagrant destroy,
+    hence /etc/exports must be accordingly cleaned-up before deleting
+    PRCI job folders in order to avoid nfs-server issues.
+    """
+    entries_deleted = []
+    try:
+        if not os.geteuid()==0:
+            print("\nUnable to prune /etc/exports. Must be run as root\n")
+            return None
+    except:
+	pass
+
+    exports = open('/etc/exports', 'r+')
+    content = exports.readlines()
+    exports.seek(0)
+    for line in content:
+        if "# VAGRANT-BEGIN" not in line:
+            exports.write(line)
+        else:
+            next_item = content.index(line) + 1
```

Comment:
we could implement this in a more efficient way but I do not want to complicate the source code for a very simple feature.

Reply:
Hey Triviño !
Looks ok now !